Compiler/Template/CodegenC.tpl

```diff
@@ -3827,7 +3809,9 @@ template evaluateDAEResiduals(list<list<SimEqSystem>> resEquations, String model
     data->simulationInfo->callStatistics.functionEvalDAE++;
 
     <%if Flags.isSet(Flags.PARMODAUTO) then 'PM_functionDAERes(<%nrfuncs%>, data, threadData, functionDAERes_systems);'
-    else '<%fncalls%>' %>
+    else (resEquations |> eqLst => (eqLst |> eq hasindex i0 =>
+                    equation_call(eq, modelNamePrefix)
+                    ;separator="\n");separator="\n") %>
```

Comment:
@sjoelund I guess this part of the patch is not correct.
with old code buffer eqns and eqFuncs were filled. In line 3820 you do not remove printing of eqFunc ( which is empty at that point, so no problem, but superfluous) Here (3830), buffer fncalls was printed - but you put content of old buffer eqns...